src/haddock/libs/libpdb.py

```diff
@@ -12,6 +12,24 @@
 from haddock.modules import working_directory
 
 
```

Comment:
What will happen if a PDB file is truncated (e.g. the lines do not have the correct official length, missing for example the segid columns?

Reply:
This is sometimes happening depending on the software used... or are you using some PDB-tools tool to first "clean" the file?

Reply:
Currently in the `main` branch, there is "sanitize" step that takes place in `topoaa` after the input molecule is split into models:

https://github.com/haddocking/haddock3/blob/2dea812673cee9fb759a5bba52f9576d4504208f/src/haddock/modules/topology/topoaa/__init__.py#L94

This sanitize step removes unneeded lines containing the [following strings](https://github.com/haddocking/haddock3/blob/2dea812673cee9fb759a5bba52f9576d4504208f/src/haddock/libs/libpdb.py#L26).

And [removes lines](https://github.com/haddocking/haddock3/blob/2dea812673cee9fb759a5bba52f9576d4504208f/src/haddock/libs/libpdb.py#L20) from residues not supported by the [topology format](https://github.com/haddocking/haddock3/blob/main/src/haddock/cns/toppar/haddock.top). Residues are removed either if `RESIdue` or `Residue` in the topology format. Have a look if that is desired.

This part of the process was inherited from `beta1` branch, so now is the time to revisit it and correct it.

Question: Should all PDBs be sanitized before entering the HADDOCK3 workflow?